sources/packages/backend/apps/api/src/testHelpers/auth/student-token-helpers.ts

```diff
@@ -0,0 +1,67 @@
+import { KeycloakService } from "../../services";
+import { TokenResponse } from "../../services/auth/keycloak/token-response.model";
+import { Student, User } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+
+const studentClientId = "student";
+
+export enum FakeStudentUsersTypes {
+  /**
+   * Fake student user type used for simple tests.
+   * This student has a valid SIN and SIN consent.
+   */
+  FakeStudentUserType1,
```

Comment:
I believe that we should decide on a pattern to have the ENV variable, user name, and FakeStudentUsersTypes in sync.
Thoughts? 
@andrepestana-aot @ann-aot @dheepak-aot @guru-aot @sh16011993

Reply:
Yes. I agree. Also IMO type1 could be named more aligned with the type.

for e.g if we differentiate student users between bceid and bcsc authorized users, then FakeStudentUserTypeBceid and FakeStudentUserBCSC 

or if we don't have such thing, then just FakeStudentUser would be better than FakeStudentUserTypeSomething.

Reply:
@dheepak-aot we'll need different student users to test other data like with PD Status or temporary SIN as test those conditions.

Reply:
how about following the same format as that of the env variable?

Reply:
> how about following the same format as that of the env variable?

Do you mean same name as the env variable?

I think we could just give them different numbers and explain what data or what they are used for in the comments. Otherwise they have too long names like `FakeUserStudentWithPdStatusXAndValidSinAndApplicationUsedToTestY`.

Reply:
To be honest, because we are talking about test users I would not mind being pretty expressive with the naming.
We do not need to express in the name absolute every single detail but some key points would be good, IMO, and then we can extend the intended scenario in the comments.
`FakeUserStudentWithPdStatusXAndValidSinAndApplicationUsedToTestY` can be, for instance, `FakeStudentusers.BCeIDWithPDAndTemporarySIN`
or even
`FakeStudentusers.BCSCWithActiveRestrictionToPreventApplicationSubmission`

We always have the possibility to mock the token authentication and have just a few real students execute the Keycloak authentication, after the real authentication is done we can then mock the validation token. This would allow us to create students as needed. It would also break a bit the idea of the E2E to validate the authentication part, something to maybe have a DEV sync up to talk about it.